Marlin/src/pins/stm32f1/pins_FLY_MINI.h

```diff
@@ -0,0 +1,190 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#pragma once
+
+#ifndef __STM32F1__
+  #error "Oops! Select an STM32F1 board in 'Tools > Board.'"
+#endif
+
+#define BOARD_INFO_NAME   "FLY_MINI"
+#define BOARD_WEBSITE_URL    "github.com/FLYmaker"
+#define DISABLE_JTAG
+
+//
+// Flash EEPROM Emulation
+//
+#define FLASH_EEPROM_EMULATION
+#define EEPROM_PAGE_SIZE     uint16(0x800) // 2KB
+#define EEPROM_START_ADDRESS uint32(0x8000000 + 256 * 1024 - 2 * EEPROM_PAGE_SIZE)
+#undef E2END
+#define E2END                (EEPROM_PAGE_SIZE - 1) // 2KB
+
+
+
+
+#define SERVO0_PIN                         PA8
+//
+// Limit Switches
+//
+#define X_MIN_PIN                           PC12
+#define X_MAX_PIN                           PC11
+#define Y_MIN_PIN                           PC10
+#define Y_MAX_PIN                           PA15
+#define Z_MIN_PIN                           PA14
+#define Z_MAX_PIN                           PA13
+
+
+
+
+//
+// Steppers
+//
+#define X_STEP_PIN                          PB1
+#define X_DIR_PIN                           PB2
+#define X_ENABLE_PIN                        PB10
+#ifndef X_CS_PIN
+  #define X_CS_PIN                          PB0
+#endif
+
+
+
+
+#define Y_STEP_PIN                          PA2
+#define Y_DIR_PIN                           PC4
+#define Y_ENABLE_PIN                        PC5
+#ifndef Y_CS_PIN
+  #define Y_CS_PIN                          PA7
+#endif
+
+
+#define Z_STEP_PIN                          PA3
+#define Z_DIR_PIN                           PA5
+#define Z_ENABLE_PIN                        PA6
+#ifndef Z_CS_PIN
+  #define Z_CS_PIN                          PA4
+#endif
+
+
+
+
+#define E0_STEP_PIN                         PA1
+#define E0_DIR_PIN                          PC3
+#define E0_ENABLE_PIN                       PA0
+#ifndef E0_CS_PIN
+  #define E0_CS_PIN                         PC2
+#endif
+
+#if ENABLED(TMC_USE_SW_SPI)
+  #ifndef TMC_SW_MOSI
+    #define TMC_SW_MOSI                     PB15
+  #endif
+  #ifndef TMC_SW_MISO
+    #define TMC_SW_MISO                     PB14
+  #endif
+  #ifndef TMC_SW_SCK
+    #define TMC_SW_SCK                      PB13
+  #endif
+#endif
+
+#if HAS_TMC_UART
+  
+  //
+  // Software serial
+  //
+  #define X_SERIAL_TX_PIN                   PB0
+  #define X_SERIAL_RX_PIN                   PB0
+
+  #define Y_SERIAL_TX_PIN                   PA7
+  #define Y_SERIAL_RX_PIN                   PA7
+
+  #define Z_SERIAL_TX_PIN                   PA4
+  #define Z_SERIAL_RX_PIN                   PA4
+
+  #define E0_SERIAL_TX_PIN                  PC2
+  #define E0_SERIAL_RX_PIN                  PC2
+
+#endif
+
+
+//
+// Heaters / Fans
+//
+#define HEATER_0_PIN                        PC6
```

Comment:
I _really_ appreciate that you arranged all the fan/heater pins on the channels of a single timer.